### PR TITLE
Apply dss to state in hybrid driver

### PR DESCRIPTION
**Pull request by dennisYatunin**
This PR applies dss to every field in the state Y for all hybrid examples that use `driver.jl`.

### CODE REVIEW

File: examples/hybrid/driver.jl [hunk 1]
```diff
@@ -120,9 +120,6 @@ else
         c = center_initial_condition.(ᶜlocal_geometry),
         f = face_initial_condition.(ᶠlocal_geometry),
     )
-    # apply dss in case the initial conditions are discontinuous across elements
-    Spaces.weighted_dss!(Y.c, comms_ctx)
-    Spaces.weighted_dss!(Y.f, comms_ctx)
```
Comment (simonbyrne) [inline examples/hybrid/driver.jl:125]: I think we may still need this, as it needs to get called every stage

Comment (dennisYatunin) [reply examples/hybrid/driver.jl:125]: `func_start = true` in the callback ensures that it gets called on the initial conditions

Comment (simonbyrne) [reply examples/hybrid/driver.jl:125]: no, i meant it needs to get called every RK stage (as opposed to every step)

Comment (dennisYatunin) [reply examples/hybrid/driver.jl:125]: I don't think there's a way to supply stage callbacks to OrdinaryDiffEq. We can only make a callback that gets called once per tilmestep.

Comment (dennisYatunin) [reply examples/hybrid/driver.jl:125]: In theory, if we dss the tendency on every RK stage, then the state will remain continuous across elements. This is apparently not true after many timestep, but applying dss at the end of every timestep should fix that.

**Review by simonbyrne — APPROVED** [review]

**Pull request merged**